uploadFile.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+import requests
+import argparse
+from pathlib import Path
+from dafni_cli.login import login
+from dafni_cli.nims import (
+    validate_model_definition,
+    get_model_upload_urls,
+    upload_file_to_minio,
+    start_model_ingest,
+    start_model_version_ingest,
+)
+
+args = sys.argv[1:]
+definition_path = Path(args[0])
+image_path = Path(args[1])
+username = args[2]
+password = args[3]
+version_message = args[4]
+parent_model = args[5] if len(args) > 5 else None
+
+print("Logging in to DAFNI")
+jwt = login(username, password)
+print(jwt)
```

Comment:
If you actually want to print this, I'd do it along with something like `print(f"Using JWT: {jwt}")` just to make it a bit clearer

Reply:
oh oops no wasn't meant to be printing that